python/hail/methods/statgen.py

```diff
@@ -394,32 +393,32 @@ def logreg(dataset, test, y, x, covariates=[], root='logreg'):
     where :math:`\mathrm{sigmoid}` is the `sigmoid function`_, the genotype
     :math:`\mathrm{gt}` is coded as 0 for HomRef, 1 for Het, and 2 for
     HomVar, and the Boolean covariate :math:`\mathrm{is_female}` is coded as
-    for true (female) and 0 for false (male). The null model sets
+    for ``True`` (female) and 0 for ``False`` (male). The null model sets
     :math:`\beta_1 = 0`.
 
     .. _sigmoid function: https://en.wikipedia.org/wiki/Sigmoid_function
 
     The resulting variant annotations depend on the test statistic as shown
     in the tables below.
 
-    ========== ================ ======= ============================================
-    Test       Field            Type    Value
-    ========== ================ ======= ============================================
-    Wald       `logreg.beta`    float64 fit effect coefficient,
-                                        :math:`\hat\beta_1`
-    Wald       `logreg.se`      float64 estimated standard error,
-                                        :math:`\widehat{\mathrm{se}}`
-    Wald       `logreg.z_stat`  float64 Wald :math:`z`-statistic, equal to
-                                        :math:`\hat\beta_1 / \widehat{\mathrm{se}}`
-    Wald       `logreg.p_value` float64 Wald p-value testing :math:`\beta_1 = 0`
-    LRT, Firth `logreg.beta`    float64 fit effect coefficient,
-                                        :math:`\hat\beta_1`
-    LRT, Firth `logreg.chi2`    float64 deviance statistic
-    LRT, Firth `logreg.p_value` float64 LRT / Firth p-value testing
-                                        :math:`\beta_1 = 0`
-    Score      `logreg.chi2`    float64 score statistic
-    Score      `logreg.p_value` float64 score p-value testing :math:`\beta_1 = 0`
-    ========== ================ ======= ============================================
+    ========== ======================= ======= ============================================
+    Test       Field                   Type    Value
+    ========== ======================= ======= ============================================
+    Wald       `logreg.beta`           float64 fit effect coefficient,
+                                               :math:`\hat\beta_1`
+    Wald       `logreg.standard_error` float64 estimated standard error,
+                                               :math:`\widehat{\mathrm{se}}`
+    Wald       `logreg.z_stat`         float64 Wald :math:`z`-statistic, equal to
+                                               :math:`\hat\beta_1 / \widehat{\mathrm{se}}`
+    Wald       `logreg.p_value`        float64 Wald p-value testing :math:`\beta_1 = 0`
+    LRT, Firth `logreg.beta`           float64 fit effect coefficient,
+                                               :math:`\hat\beta_1`
+    LRT, Firth `logreg.chi_squared`    float64 deviance statistic
```

Comment:
use `chi_sq_stat` or `chi_squared_stat` to match `t_stat` and `z_stat`?

Reply:
good call